configure.ac

```diff
@@ -1433,6 +1433,10 @@ if test "$enable_libraries" = "yes" -a "$have_glut" = yes -a "$have_jpeg" = 1; t
   configured_to_build="${configured_to_build} libraries_graphics"
 fi
 
+if test "${enable_manager}" = yes -a "$have_Xcb" = yes -a "$have_Xcbatom" = yes; then
+  configured_to_build="${configured_to_build} manager_screen_saver"
```

Comment:
This is already present in this file: https://github.com/BOINC/boinc/blob/a9866080c5f9854530df5fae7f8827e6e5fffe67/configure.ac#L1098-L1102

Reply:
Yes, I check this conditions again and just add a nice msg to the user that it know that it going to build a screen server. like I did to graphic libs. the same way.

Reply:
We have no 'Manager screensaver'. Either name it 'Client Screensaver' (the same way the folder is named) or 'X11 Screensaver' as it named in the scripts

Reply:
I name it on the condition. It can build only when you build the manager.

Reply:
Yeah, my bad: 'apps_gui' should be renamed as well to avoid misleading

Reply:
I can change it to `manager_clientscr`